index.html

```diff
@@ -0,0 +1,1367 @@
+          <h5 class="pl-2 text-body font-weight-bold title-15 line-h-18 hover-red">What Netflix's 'The Spy' Gets Wrong About Eli Cohen</h5>
+        </div>
+      </div>
+    
+    </div>
+    
+  
+  </div> <!--end 9 container-->
+    
+
+  <!-- 10 container -->
+  <!-- Sports content -->
+  <div class="container-fluid border-top border-bottom border-light pb-3">
+    <div class="row pt-3">
+        <div class="">
+          <h5 class="text-uppercase font-weight-bold c-pointer red-color pl-4">Sports</h5>
+        </div>
+    </div>
+  
+    <div class="row">       
+  
+      <div class="card col-12-sm col-md-4 col-lg-4 rounded-0 border border-white">
+        <img class="card-img-top rounded-0" src="assets/images/sp1.jpg" alt="#">
+        <h5 class="title-gen-1 text-body font-weight-bold">Boston Police Commissioner Defends Officers After Clashes at Straight Pride</h5>
+      </div>
+  
+      <div class=" card col-12-sm col-md-4 col-lg-4 rounded-0 border border-white">
+        <img class="card-img-top rounded-0" src="assets/images/sp2.jpg" alt="#">
+        <h5 class="title-gen-1 text-body font-weight-bold">Solheim Cup 2019 Teams: Full Team USA and Team Europe Rosters</h5>
+      </div>
+  
+      <div class=" card col-12-sm col-md-4 col-lg-4 d-flex flex-column rounded-0 border border-white">
+        <div class="d-flex flex-row border-bottom border-secondary py-2 rounded-0">
+          <img class="card-img-top rounded-0  img-right-1" src="assets/images/sp3.jpg" alt="#">
+          <h5 class="pl-2 text-body font-weight-bold title-15 line-h-18 hover-red">FIBA World Cup Draw: Quarterfinals Schedule, Live Stream and Latest Odds</h5>
+        </div>
+  
+        <div class="d-flex flex-row border-bottom border-secondary py-2 rounded-0">
+          <img class="card-img-top rounded-0 img-right-1 " src="assets/images/sp4.jpg" alt="#">
+          <h5 class="pl-2 text-body font-weight-bold title-15 line-h-18 hover-red">USMNT vs. Uruguay: Where to Watch, Live Stream, TV Channel, Odds</h5>
+        </div>
+  
+        <div class="d-flex flex-row py-2 border border-white">
+          <img class="card-img-top rounded-0 img-right-1 rounded-0" src="assets/images/sp5.jpg" alt="#">
+          <h5 class="pl-2 text-body font-weight-bold title-15 line-h-18 hover-red">Monday Night RAW Results (9/9)</h5>
+        </div>
+      </div>
+  
+    </div>
+  
+  </div> <!--end 10 container-->
+
+
+  <!--11 container-->
+  <!--subscribe newsletter-->
+  <div class="container mb-3 mt-4">
+    <div class="row">
+      <!--left content-->
+      <div class="col-lg-8 col-sm-12">
+        <div class="text-uppercase font-weight-bold red-color">Subscribe</div> <!--title-->
+        <div class="d-flex flex-row bg-light"> <!--to put content in a row-->
+          <div class="subs-left height-mg"> <!--magazines-->
+            <img class="pt-3 pl-3" src="assets/images/mg02.jpg" alt="images">
+            <img  class="mg-herb"src="assets/images/mg01.jpg" alt="images">
```

Comment:
You need spacing for the src attribute :smiley:

Reply:
Thank you for the feedback, Samgaco  :) I did all the changes required and used the validator correctly.

Next time I will pay attention to ''meaningful names for the Pull Requests'' and will ''add a short summary'', so thanks for the advice.